src/pages/Home/home.js

```diff
@@ -1,7 +1,43 @@
-export const Home = () => {
-  return (
-    <>
-      New Home
-    </>
-  )
-}
\ No newline at end of file
+import axios from "axios";
+import React from "react";
+
+export class Home extends React.Component {
+  state = {
+    data: {},
```

Comment:
Should this be initialized to null instead of an empty object?

Reply:
I'll ask Cristian because I am not 100% either.

Reply:
are you going to fetch data here? I would set it to null because it makes life easier (for now at least)

Reply:
I tried changing it to null and the render failed. If I keep it as an empty object it displays the results of my API request.

Reply:
Yes I also have this problem with **null** and I read / heard somewhere to just set it to **empty [] / {}** until you receive your data